Saturation/ProvingHelper.cpp

```diff
@@ -57,9 +57,6 @@ using namespace Shell;
   catch(const std::bad_alloc &) {
     env.statistics->terminationReason=Statistics::MEMORY_LIMIT;
     env.statistics->refutation=0;
-    size_t limit=Lib::getMemoryLimit();
-    //add extra 1 MB to allow proper termination
-    Lib::setMemoryLimit(limit+1000000);
```

Comment:
This doesn't work because `setrlimit` cannot increase the limit once it is lowered. *However*, it doesn't seem to be actually necessary as we free some things while the exception is in flight.